libcst/codemod/commands/add_trailing_commas.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import textwrap
+from typing import Dict, Optional
+
+import libcst as cst
+from libcst.codemod import CodemodContext, VisitorBasedCodemodCommand
+
+
+presets_per_formatter: Dict[str, Dict[str, int]] = {
+    "black": {
+        "parameter_count": 1,
+        "argument_count": 3,
```

Comment:
this `3` feels arbitrary, but whatever :)

Reply:
Yeah. I'm pretty sure 1 is too aggressive on function calls, I was torn between 2 and 3.